webhookbot/webhookbot/handler.go

```diff
@@ -139,6 +159,51 @@ func (h *Handler) handleCreate(cmd string, msg chat1.MsgSummary) (err error) {
 	return nil
 }
 
+func (h *Handler) handleUpdate(cmd string, msg chat1.MsgSummary) (err error) {
+	convID := msg.ConvID
+	toks := strings.Split(cmd, " ")
+	if len(toks) < 3 {
+		h.ChatEcho(convID, "invalid number of arguments, must specify a name")
+		return nil
+	}
+	err = h.checkAllowed(msg)
+	switch err {
+	case nil:
+	case errNotAllowed:
+		h.ChatEcho(convID, err.Error())
+		return nil
+	default:
+		return err
+	}
+	h.stats.Count("update")
+	name := toks[2]
+	// template is whatever remains after removing "!webhook create <name>", and trimming spaces.
```

Comment:
i'm not sure it makes sense to include this part that cleans out the `!webhook create` and the hook name from the received message since the helper will be called from the update command as well, but it definitely makes sense to put the validation into a helper func. let me know if you disagree and we can discuss

Reply:
the cleaning code can just be parameterized to work for both